TriblerGUI/widgets/homepage.py

```diff
@@ -108,4 +107,4 @@ def on_home_page_item_clicked(self, row, col):
             channel_info = cell_widget.channel_info
             self.window().channel_page.initialize_with_channel(channel_info)
             self.window().navigation_stack.append(self.window().stackedWidget.currentIndex())
-            self.window().stackedWidget.setCurrentIndex(PAGE_CHANNEL_DETAILS)
```

Comment:
Why is this commented out? Should have a TODO or a FIXME.

Reply:
Fixed